proto/cachepubsub.proto

```diff
@@ -70,13 +70,20 @@ message _SubscriptionRequest {
   // The literal topic name to which you want to subscribe.
   string topic = 2;
 
-  // --> Providing this is not required. <--
-  //
   // If provided, attempt to reconnect to the topic and replay messages starting from
   // the provided sequence number. You may get a discontinuity if some (or all) of the
   // messages are not available.
+  // If provided at 1, you may receive some messages leading up to whatever the
+  // newest message is. The exact amount is unspecified and subject to change.
```

Comment:
Will topic sequence number `1` ever be sent as a `resume_at_topic_sequence_number`? When would a client provide this number?

Reply:
It's used routinely by users of http long polling (ask khawaja, he's a big fan of resume at 1)